platform/server/src/resolvers/custom-resolvers.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+    Location: {
```

Comment:
This is needed as ApolloServer seems to be complaining about Location type missing a `__resolveType`

a.k.a without this I had `Type "Location" is missing a "__resolveType" resolver interface GraphQl`. 

Can graphback take care of that instead?

@wtrocki @craicoverflow

Reply:
There is flag to disable resolve type checks that we use in templates

Reply:
Okay, how do we do that?

Reply:
```
const schema = makeExecutableSchema({
  typeDefs,
  resolvers,
  resolverValidationOptions: { requireResolversForResolveType: false },
});
```

Reply:
Yes, that could also too. In fact, it was among the explored options before I went with the current option. 

With what you are proposing, it seems like one has to do "manually" while what I was asking is if `graphback` can automatically generate the part I added (also if it seems reasonable?).